src/webstore/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @author acvetkov@yandex-team.ru
+ * @overview
+ */
+
+import TokenManager from '../token/index';
+import * as api from './items';
+
+export default class {
+
+    /**
+     * @param {String} code
+     * @param {String} clientId
+     * @param {String} clientSecret
+     */
+    constructor (code, clientId, clientSecret) {
+        this.tokenManager = new TokenManager(code, clientId, clientSecret);
+    }
+
+    /**
+     * Publish item in chrome web store
+     * @param {String} itemId
+     * @returns {Promise<ChromeStorePublishInfo>}
+     */
+    publish (itemId) {
+        return this.tokenManager.get()
+            .then(token => {
+                return api.publish(token, itemId);
+            });
+    }
+
+    /**
+     * Create new item
+     * @param {Blob} content
+     * @returns {ChromeStoreItem<ChromeStorePublishInfo>}
```

Comment:
sorry, I thought it should be `<ChromeStoreItemChromeStorePublishInfo>`
forget it :)